src/dashboard/components/dashboards_list_component.py

```diff
@@ -0,0 +1,77 @@
+"""Dashboards list component."""
+
+from datetime import datetime, timedelta
+
+from dash.dependencies import Component
+
+from dashboard.models.user import Dashboard
+
```

Comment:
This is just nitpicking, but there shouldn't be an empty line here.
PEP8 specifies grouping imports into 3 groups with one empty line between: standard lib imports, third party lib imports, imports from project modules.

Reply:
I'm not sure why, but isort is the cause of this and linting fails unless it is like this.